### PR TITLE
queryStream* one-shot operation

### PR DESCRIPTION
**Pull request by ruslansennov**

### CODE REVIEW

File: src/main/java/io/vertx/ext/sql/RowStreamWrapper.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package io.vertx.ext.sql;
```
Comment (pmlopes) [inline src/main/java/io/vertx/ext/sql/RowStreamWrapper.java]: It misses the usual header with the copyright and license

File: src/main/java/io/vertx/ext/sql/RowStreamWrapper.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonArray;
+
+import java.util.List;
+
+/**
+ * @author <a href="mailto:ruslan.sennov@gmail.com">Ruslan Sennov</a>
```
Comment (pmlopes) [inline src/main/java/io/vertx/ext/sql/RowStreamWrapper.java]: Maybe a small doc with the use case

Comment (pmlopes) [reply src/main/java/io/vertx/ext/sql/RowStreamWrapper.java]: I think this should be in the `impl` as it’s a wrapper that auto closes the connection not really an api to be used by the end user, right?

**Review by pmlopes — CHANGES_REQUESTED** [review]

File: src/main/java/io/vertx/ext/sql/RowStreamWrapper.java (hunk 2 above)
Comment (ruslansennov) [reply src/main/java/io/vertx/ext/sql/RowStreamWrapper.java]: Yes, this is the wrapper, it has package-visibility and is not intended for use by anyone. I see no difference if we move it to `impl` package or leave it here with  package-visibility. But if it is important, I can move it :)

Comment (pmlopes) [reply src/main/java/io/vertx/ext/sql/RowStreamWrapper.java]: I understand but we do this as convention to explicitly tell users that internal APIs can change at any time and what’s in the main package is stable

Comment (ruslansennov) [reply src/main/java/io/vertx/ext/sql/RowStreamWrapper.java]: thank you, done

**Review by pmlopes — APPROVED** [review]

**Pull request merged**